helm/conjur-config-cluster-prep/templates/golden_configmap.yaml

```diff
@@ -1,3 +1,5 @@
+# The Golden ConfigMap keeps a reference copy of Conjur configuration information that will be used for subsequent operations such as preparing application Namespaces for using Conjur Kubernetes authentication.
+
```

Comment:
Probably neater to not include empty lines after your comments, since these empty lines will appear in the rendered manifests.